src/main/java/com/redhat/devtools/lsp4ij/client/features/EditorBehaviorFeature.java

```diff
@@ -0,0 +1,132 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at https://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+
+package com.redhat.devtools.lsp4ij.client.features;
+
+import com.intellij.psi.PsiFile;
+import com.redhat.devtools.lsp4ij.LanguageServiceAccessor;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Client-side editor behavior feature. This does not correspond to an actual LSP feature.
+ */
+@ApiStatus.Experimental
+public class EditorBehaviorFeature {
```

Comment:
Note that I've named this `EditorBehaviorFeature` instead of `LSPEditorBehaviorFeature` because, unlike all of the other features, it's not based on LSP. However, having it be centrally available via `LSPClientFeatures` still makes sense. Let me know if you'd prefer a different name, though.